root/cgi-bin/print_ok.py

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)